ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -39,13 +39,13 @@ TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-
 
 set +e
 wait "$!"
-ret=$?
+ret="$?"
 set -e
 
+sed 's/password: .*/password: REDACTED/' "${dir}/.openshift_install.log" >"${ARTIFACT_DIR}/.openshift_install.log"
+cp "${dir}"/log-bundle-*.tar.gz "${ARTIFACT_DIR}/" 2>/dev/null || :
```

Comment:
Let's not use two mechanisms to ignore the output of something in one shell script, if we can.

Reply:
> Let's not use two mechanisms to ignore the output of something in one shell script, if we can.

The `|| :` is from 20157f66e1 (#7899), but sure, I've pushed b3f7fba09 to shift this `cp` up into the `set +e` block.